libs/ha/ha_common_libs.py

```diff
@@ -314,6 +317,7 @@ def host_safe_unsafe_power_off(self, host: str, bmc_obj=None,
         :param is_safe: Power off host with safe/unsafe shutdown
         :rtype: boolean from polling_host() response
         """
+        LOGGER.info("Shutting down %s with is_safe method set to %s", host, is_safe)
```

Comment:
do we need to mention this is_safe method?

Reply:
> do we need to mention this is_safe method?

My thought is, in test function logger we were mentioning safe or unsafe keyword. But as this will be common statement for both the opetion, thought to add the same to differentiate the same. Let me know your viewes.

Reply:
What u think @ApurwaMohite 
Was just wondering will it create confusion instead?

Reply:
Yes, it will create confusion. 
I guess, instead of this you can add logger inside if-else e.g. Safe shutdown or Unsafe Shutdown.